### ISSUE TITLE
Presets Panel: Keyboard Access Not Provided for Icon Links 

### ISSUE THREAD

**Issue by derekjackson-das** [danny-avila/LibreChat#5451]
## Issue Description
The icon links that are revealed on hover are not accessible via keyboard or screen readers. This creates a barrier for users who rely on assistive technologies or keyboard navigation.

Related to:
-  #5450 
- #5452 
- #5453 
- #5489 

Derived from #5187 .

## WCAG Criteria
[1.3.1 Info and Relationships - A](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships)

## Notes
The simplest solution, which is a small change to the UI:
- Remove the hover effect so that the icons are persistent.
- Once the icons are always visible, they can then be reached with the keyboard.

## Screenshot or Screen Recording
[buttons inaccessible to keyboard Screenshot or Screen Recording](https://share.zight.com/geuDpOB4)

[Persistent icons demo - Screen Recording](https://share.zight.com/o0u179lZ)

## Relevant Code
```html
<div role="option" aria-selected="false" aria-label="New Preset:" data-testid="preset-item-[object Object]" class="group m-1.5 flex cursor-pointer gap-2 rounded px-5 py-2.5 !pr-3 text-sm !opacity-100 hover:bg-black/5 focus:ring-0 radix-disabled:pointer-events-none radix-disabled:opacity-50 dark:hover:bg-gray-600 md:min-w-[240px]" tabindex="0" ... >
```
